main.js

```diff
@@ -112,6 +112,7 @@ if (!fs.existsSync('./config.js')) {
 global.Data = {
 	Movedex: require('./data/movedex.js').movedex,
 	Abilitydex: require('./data/abilitydex.js').abilitydex,
+	CapNames: require('./data/capnamelist.js').capnamelist, //List of CAP prefixes/suffixes for random names
```

Comment:
I'd kinda prefer this object to be named `Namelist`. While it's only used in the cap command it's definitely potentially useful for more, and I think it's a more concise name. The export itself was done correctly, I just would rather have `Namelist: require('./data/namelist.js').namelist,`